verticaltabs.js

```diff
@@ -697,6 +666,15 @@ VerticalTabs.prototype = {
       }
     }, 150);
 
+    function checkDevTheme() {
+      if (/devedition/.test(mainWindow.style.backgroundImage)) {
```

Comment:
hehe, I think so. it's the only way I could find from before when we had the sizing issues